File: Lista_5/Zad2+3/Python/Zad3.py
import time

MYFILENAME = '../Covid19.txt'


if __name__ == '__main__':
    my_file = open(MYFILENAME, "r")

    sum_ = 0
    country = input('Write your country: ')
    start_time = time.time()

    for line in my_file:
        word_list = line.split()

        if len(word_list) >= 7 and word_list[6] == country:
            sum_ += int(word_list[4]) if word_list[4].isdigit() else 0

    print('Total cases: %d' % sum_)
    print('Exec time: %f' % (time.time() - start_time))

File: Lista_5/Zad2+3/Java/src/Zad3.java
import java.io.File;
import java.util.Scanner;

public class Zad3 {

    public static void main(String[] args) {
        Scanner inputWordScanner = new Scanner(System.in);
        System.out.println("Write your country: ");
        String inputWord = inputWordScanner.next();

        long startTime = System.nanoTime();

        try {
            Scanner fileScanner = new Scanner(new File("../Covid19.txt"));
            int sum = 0;

            while (fileScanner.hasNext()) {
                String fileLine = fileScanner.nextLine();
                String[] lineArray = fileLine.split("\t| ");

                if (lineArray.length > 6 && lineArray[6].equals(inputWord))
                    try {
                        sum += Integer.parseInt(lineArray[4]);
                    } catch (Exception ignored) {}

            }

            System.out.println("Sum " + sum);
            System.out.println("Exec time: " + ((System.nanoTime() - startTime) / 1000000) + "ms");
        } catch (Exception ex) {
            System.out.println("Cannot find the file !");
        }
    }

}
